.github/workflows/build_workflow.yml

```
@@ -19,7 +19,7 @@ jobs:
     timeout-minutes: 2
     steps:
       - id: skip_check
-        uses: fkirc/skip-duplicate-actions@master
+        uses: fkirc/skip-duplicate-actions@main
```

```suggestion
        uses: fkirc/skip-duplicate-actions@master
```
The branch reference here shouldn't be updated since it is for another repo.